plugins/globalias/globalias.plugin.zsh

```diff
@@ -4,8 +4,11 @@ globalias() {
    zle self-insert
 }
 zle -N globalias
-bindkey -e " " globalias
-bindkey -v " " globalias
-bindkey -e "^ " magic-space           # control-space to bypass completion
-bindkey -v "^ " magic-space
-bindkey -M isearch " " magic-space # normal space during searches
```

Comment:
It must be this from the [Zsh Line Editor docs](http://zsh.sourceforge.net/Doc/Release/Zsh-Line-Editor.html#index-local-keymaps):

> While for normal editing a single keymap is used exclusively, in many modes a local keymap allows for some keys to be customised. For example, in an incremental search mode, a binding in the isearch keymap will override a binding in the main keymap but **all keys that are not overridden can still be used**.